sdk/keyvault/azure-keyvault-certificates-apis/azure/keyvault/certificates/models.pyi

```diff
@@ -0,0 +1,212 @@
+    @property
+    def cer(self) -> bytes: ...
+
+class CertificateOperation:
+    def __init__(self, cert_operation_id: Optional[str]=..., issuer_name: Optional[str]=..., certificate_type: Optional[str]=..., certificate_transparency: Optional[bool]=..., csr: Optional[bytes]=..., cancellation_requested: Optional[bool]=..., status: Optional[str]=..., status_details: Optional[str]=..., error: Optional[models.Error]=..., target: Optional[str]=..., request_id: Optional[str]=...) -> None: ...
+    @property
+    def id(self) -> str: ...
+    @property
+    def name(self) -> str: ...
+    @property
+    def issuer_name(self) -> str: ...
+    @property
+    def certificate_type(self) -> str: ...
+    @property
+    def certificate_transparency(self) -> bool: ...
+    @property
+    def csr(self) -> bytes: ...
+    @property
+    def cancellation_requested(self) -> bool: ...
+    @property
+    def status(self) -> str: ...
+    @property
+    def status_details(self) -> str: ...
+    @property
+    def error(self) -> models.Error: ...
+    @property
+    def target(self) -> str: ...
+    @property
+    def request_id(self) -> str: ...
+
+class CertificatePolicy:
+    def __init__(self, issuer_name: str, subject_name: str, exportable: Optional[bool]=..., key_type: Optional[KeyType]=..., key_size: Optional[str]=..., reuse_key: Optional[bool]=..., curve: Optional[KeyCurveName]=..., ekus: Optional[list[str]]=..., key_usage: Optional[list[Union[KeyUsageType, str]]]=..., content_type: Optional[Union[SecretContentType, str]]=..., validity_in_months: Optional[int]=..., lifetime_actions: Optional[list[LifetimeAction]]=..., certificate_type: Optional[str]=..., certificate_transparency: Optional[bool]=..., **kwargs: Any) -> None: ...
+    @property
+    def id(self) -> str: ...
+    @property
+    def exportable(self) -> bool: ...
+    @property
+    def key_type(self) -> KeyType: ...
+    @property
+    def key_size(self) -> int: ...
+    @property
+    def reuse_key(self) -> bool: ...
+    @property
+    def curve(self) -> KeyCurveName: ...
+    @property
+    def ekus(self) -> list[str]: ...
+    @property
+    def key_usage(self) -> list[KeyUsageType]: ...
+    @property
+    def content_type(self) -> SecretContentType: ...
+    @property
+    def subject_name(self) -> str: ...
+    @property
+    def san_emails(self) -> list[str]: ...
+    @property
+    def san_dns_names(self) -> list[str]: ...
+    @property
+    def san_upns(self) -> list[str]: ...
+    @property
+    def validity_in_months(self) -> int: ...
+    @property
+    def lifetime_actions(self) -> list[LifetimeAction]: ...
+    @property
+    def issuer_name(self) -> str: ...
+    @property
+    def certificate_type(self) -> str: ...
+    @property
+    def certificate_transparency(self) -> bool: ...
+    @property
+    def enabled(self) -> bool: ...
+    @property
+    def not_before(self) -> datetime: ...
+    @property
+    def expires(self) -> datetime: ...
```

Comment:
expires_on - I'll not keep repeating this one. All datetime fields should be suffixed with "on", except for "not_before" and "scheduled_purge_date".